src/api_server.h

```diff
@@ -117,6 +109,36 @@ class MetadataAgent {
   mutable std::mutex metadata_mu_;
   // A map from MonitoredResource to (JSON) resource metadata.
   std::map<MonitoredResource, Metadata> metadata_map_;
+};
+
+// Runs the metadata tasks.
+class MetadataAgent {
+ public:
+  using Metadata = MetadataStore::Metadata;
```

Comment:
This doesn't seem to be used anywhere in the MetadataAgent class. Can we remove?

Reply:
NM. I see you remove this in a later commit.